backend/src/models/user.ts

```diff
@@ -16,6 +16,16 @@ import {
 import { BCRYPT_WORK_FACTOR } from "../config";
 import { UserData } from "../interfaces/UserData";
 
+interface updateUserData {
```

Comment:
Capitalize "u" so it's `UpdateUserData`

Reply:
Same answer as: https://github.com/diogo-botelho/busk-app/pull/27#discussion_r1148089076